src/main/scripts/addon.bat

```diff
@@ -132,7 +133,7 @@ set JAVA_OPTS=%JAVA_OPTS% --add-opens=java.base/sun.net.www.protocol.https=ALL-U
 set JAVA_OPTS=%JAVA_OPTS% --add-opens=java.base/sun.net.www.protocol.jar=ALL-UNNAMED
 set JAVA_OPTS=%JAVA_OPTS% --add-opens=java.base/sun.nio.ch=ALL-UNNAMED
 set JAVA_OPTS=%JAVA_OPTS% --add-opens=java.base/sun.security.util=ALL-UNNAMED
-%_RUNJAVA% %JAVA_OPTS% -jar "%PLF_HOME%\addons\addons-manager.jar" %CMD_LINE_ARGS%
+%_RUNJAVA% %JAVA_OPTS% %EXO_AM_OPTS% -jar "%PLF_HOME%\addons\addons-manager.jar" %CMD_LINE_ARGS%
```

Comment:
no need of this change, the value of `EXO_AM_OPTS` was already inserted in `JAVA_OPTS` in the previous line

```suggestion
%_RUNJAVA% %JAVA_OPTS% -jar "%PLF_HOME%\addons\addons-manager.jar" %CMD_LINE_ARGS%
```